### PR TITLE
Add table sorting to the release table

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Upgraded to Vanilla 2.0.1
- Add the Vanilla sortable pattern to the table

## QA
- Wait for the deployed to staging to populate
- Then sort by clicking the table heading
- Check its in the correct order

Fixes https://github.com/canonical-web-and-design/releases.demo.haus/issues/12

### CODE REVIEW

**Comment by nottrobin** [comment]
It seems to me, the whole point of doing this would be to default to it being sorted by most recently deployed first. Can we do that?

File: templates/index.html [hunk 1]
```diff
@@ -3,337 +3,467 @@
...
+      .then(writeDomainDetails(domainRow))
+    }
+    )
+  </script>
 
-        <div class="p-strip is-shallow is-bordered">
-          <div class="row">
-            <h3>Staging domains not yet supported</h3>
+  <script>
+    (function () {
```
Comment (nottrobin) [inline templates/index.html]: Wouldn't it be nice if these were modules?

**Comment by anthonydillon** [comment]
I think Python should return the initial sorted order to be rendered and use the JS to add additional sorting.

File: templates/index.html (hunk 1 above)
Comment (anthonydillon) [reply templates/index.html]: This would be nice but doesn't get use much as we have no build pipeline or tests.

**Pull request merged**